electricitymap/contrib/capacity_parsers/CL.py

```diff
@@ -0,0 +1,70 @@
+from datetime import datetime
+from logging import getLogger
+from typing import Dict, Union
+
+import pandas as pd
+from bs4 import BeautifulSoup
+from requests import Response, Session
+
+from electricitymap.contrib.config import ZoneKey
+
+logger = getLogger(__name__)
```

Comment:
Should we set log level here? I don't see any logger output otherwise, but might be because I'm running the script in the wrong way (`poetry run python CL.py`) 😅 
```suggestion
logger = getLogger(__name__)
basicConfig(level=INFO)
```
(above suggestion requires `from logging import basicConfig, getLogger, INFO`)